admin_manual/release_notes.rst

```diff
@@ -2,8 +2,74 @@
 Release Notes
 =============
 
-Changes in 10.0
----------------
+Changes in 10.0.1
+-----------------
+
+Hello ownCloud administrator, please read carefully to be prepared for updates and operations of your ownCloud setup.
+
+* **A new update path:** ownCloud 10.0.1 contains migration logic to allow upgrading directly from 9.0 to 10.0.1.
+* **Marketplace:** Please create an account for `the new marketplace`_. 
+  Access to optional ownCloud extensions and enterprise apps will be provided by the marketplace from now on.
+  Currently some apps are still shipped with the tarballs / packages and will be moved to the marketplace in the near future.
+* **Apps:** *LDAP*, *gallery*, *activity*, *PDF viewer*, and *text editor* were moved to the marketplace.
+* **Updates with marketplace:** During the upgrade, enabled apps are also updated by fetching new versions directly from the marketplace. If during an update, sources for some apps are missing, and the ownCloud instance has no access to the marketplace, the administrator needs to disable these apps or manually download and provide the apps before updating.
+* **App updates:** Third party apps are not disabled anymore when upgrading.
+* **Upgrade migration test:** The upgrade migration test has been removed; see :ref:`migration_test_label`. (Option ``--skip-migration-tests`` removed from update command).
+
+Settings
+~~~~~~~~
+
+* **Settings design:** Admin, personal pages, and app management are now merged together into a single "Settings" entry.
+* **Disable users:** The ability to disable users in the user management panel has been added.
+* **Password Policy:** Rules now apply not only to link passwords but also to user passwords.
+
+Infrastructure
+~~~~~~~~~~~~~~
+
+* **Client:** You need to update to `the latest desktop client version`_ (2.3).
```

Comment:
Too late to the party but thats currently not clear as oC 10.0.0/10.0.1 has a min. requirement of 2.0.0: https://github.com/owncloud/documentation/issues/3079